starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,59 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
+ function Chronometer() {
+    this.currentTime = 1;
+    this.intervalId = 0; 
+    this.minutes= "";   
+    this.seconds= ""; 
+    this.milliseconds = "";
+ }
+
+ Chronometer.prototype.startClick = function () {
+    
+    this.intervalId=setInterval(function(){
+        this.intervalId = setInterval (function(){
+            this.currentTime +=1;
```

Comment:
este this currentTime le hace falta un contexto, es necesario hacer in bind(this) o declarar una variable `var that = this`  para que cuando se use dentro de otras funciones el ordena sepa que te estas refiriendo a ese `that.currentTime.` En este articulo explican muy bien este tema [https://codeburst.io/understanding-that-and-bind-8778f779b149](url)